index.html

```diff
@@ -9,44 +9,266 @@
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="portfolio-section2">
+    <div class="about-section">
+      <h1 class="about-title">About me</h1>
+      <p class="about-par"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hesitate to contact me.</p>
+      <button class="btn1">Get my resume</button>
+    </div>
+
+    <div class="all-about">
+<!-- card -->
+<div class="card about-card">
+  <img class="about-img" src="img3.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">languages</h2>
+    <ul class="skills">
+      <li class="skill skill-about">JavaScript</li>
+      <li class="skill skill-about">Ruby</li>
+      <li class="skill skill-about">HTML</li>
+      <li class="skill skill-about">CSS</li>
+    </ul>
+    
+  </div>
+</div>
+ <!-- card -->
+ <div class="card about-card">
+  <img class="about-img" src="img1.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">languages</h2>
+    <ul class="skills">
+      <li class="skill skill-about">JavaScript</li>
+      <li class="skill skill-about">Ruby</li>
+      <li class="skill skill-about">HTML</li>
+      <li class="skill skill-about">CSS</li>
+    </ul>
+    
+  </div>
+</div>
+ <!-- card -->
+ <div class="card about-card">
+  <img class="about-img" src="img2.png" alt="project">
+  <div class="card-content about-content">
+    <h2 class="card-title">languages</h2>
+    <ul class="skills">
+      <li class="skill skill-about">JavaScript</li>
+      <li class="skill skill-about">Ruby</li>
+      <li class="skill skill-about">HTML</li>
+      <li class="skill skill-about">CSS</li>
+    </ul>
+    
+  </div>
+</div> 
+
+<div class="contact-1">
+  <div class="contact-full">
+    <div class="contact-para">
+      <h1 class="s">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+    </div>
+      <div class="contact-mail">
+        <form action="https://formspree.io/f/xjvqvjew" method="POST">
+          <div class="input-none">
+            <input class="input-form input-name" maxlength="30" type="text" name="name" placeholder="Full name" required>
+          </div>
+         
+          <input class="input-form input-name" maxlength="30" type="text" name="name" placeholder="Full name" required>
+          <input class="input-form email-input" type="email" name="email" placeholder="email adress"  required>
+          <textarea class="text-name" maxlength="500" type="text" name="message"  cols="30" rows="10" placeholder="Hey i'd like to connect to.." required></textarea>
+    
+        <div class="d-btn">
+          <button class=" btn2" type="submit">Get in touch</button>
+        </div>
+        </form>
```

Comment:
- [ ] Awesome work on your form and footer sections! I wanted to give you some feedback regarding the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). **It would be fantastic if you could ensure that you stick to the [given template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&mode=design&t=5aZE4bR5vlnXjyO2-0)**. 

- [ ] Kindly check the size of your "Get in touch" button, and the font color of your email to match the template.
 
- [ ] Additionally, I noticed tha**t the background image for your contact form is missing.** It would be really nice if you could add it. Thanks so much!

   ### Your design:
   ![image](https://github.com/RolandAmini/portfolio-mobile/assets/107952724/c3a78119-aa10-4573-83a4-9f914e853719)

   ### Figma design:
   ![image](https://github.com/RolandAmini/portfolio-mobile/assets/107952724/1d27025f-8e62-451f-afa8-52f7d1f9d60a)